tests/negtelnetserver.py

```diff
@@ -34,7 +33,7 @@ def telnetserver(options):
     if options.pidfile:
         pid = os.getpid()
         with open(options.pidfile, "w") as f:
-            f.write(b"{0}".format(pid))
+            f.write(str(pid))
```

Comment:
Should this not be
```
f.write(str(pid).encode("ascii"))
```
as f.write should only take bytes, right?

Reply:
If the file is opened as "w" then str is required; if opened as "wb" then bytes.